scipy/optimize/nonlin.py

```diff
@@ -1,4 +1,4 @@
-r"""
+"""
```

Comment:
this rogue `r` does this not effect the code in any way?

Reply:
Have a look at this one https://stackoverflow.com/questions/2081640/what-exactly-do-u-and-r-string-flags-do-and-what-are-raw-string-literals

Reply:
It indicates that the docstring is a Python raw string.